tests/QueryType/Update/RequestBuilderTest.php

```diff
@@ -134,6 +134,23 @@ public function testBuildAddXmlMultivalueField()
         );
     }
 
+    public function testBuildAddXmlMultivalueFieldWithEmptyArray()
+    {
+        $command = new AddCommand();
+        $command->addDocument(new Document(['id' => [1, 2, 3], 'text' => []]));
+
+        $this->assertSame(
+            '<add>'.
+            '<doc>'.
+            '<field name="id">1</field>'.
+            '<field name="id">2</field>'.
+            '<field name="id">3</field>'.
+            '</doc>'.
```

Comment:
What will be the result of ```'text' => ['']```?

It is a valid pattern to index "empty" fields because it influences the scoring. There's a factor calculated from all documents that contain a field, regardless if it is empty.

I think we must provide a way to index this:
```
<doc>
            <field name="id">1</field>
            <field name="id">2</field>
            <field name="id">3</field>
            <field name="text"></field>
</doc>

Reply:
It already worked. Added a unit test specifically for empty strings.